journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/EventualJournal.scala

```diff
@@ -356,7 +356,7 @@ object EventualJournal {
         def offset(topic: Topic, partition: Partition): F[Option[Offset]] = {
           self
             .offset(topic, partition)
-            .handleErrorWith { a =>
+            .handleErrorWith { (a) =>
```

Comment:
is it scala3 requirement? Can we change scalafmt config somehow to avoid braces for single untyped argument? I strongly disagree with such style

Reply:
https://docs.scala-lang.org/scala3/guides/migration/incompat-syntactic.html#parentheses-around-lambda-parameter - that is the syntax change

Reply:
The doc says

> When followed by its type, the parameter of a lambda is now required to be enclosed in parentheses. The following piece of code is invalid.

I see no type here thus expect to have to braces

Reply:
indeed - reverted